src/win/powershell.js

```diff
@@ -13,12 +13,11 @@ function escapeForInterpolation(arg) {
   return arg
     .replace(/[\0\u0008\u001B\u009B]/gu, "")
     .replace(/`/gu, "``")
-    .replace(/\$/gu, "`$$")
     .replace(/\r(?!\n)/gu, "")
     .replace(/\r?\n/gu, " ")
+    .replace(/(["$&'(),;{|}‘’‚‛“”„])/gu, "`$1")
     .replace(/(?<=^|[\s\u0085])([*1-6]?)(>)/gu, "$1`$2")
     .replace(/(?<=^|[\s\u0085])([#\-:<@\]])/gu, "`$1")
-    .replace(/(["&'(),;{|}‘’‚‛“”„])/gu, "`$1")
```

Comment:
This is really the only meaningful change here...

I will close this Pull Request and include this particular change in #910 instead.